doc/rst/language/spec/ranges.rst

```diff
@@ -961,8 +961,8 @@ numbers.
 
 .. _Predefined_Range_Functions:
 
-Predefined Routines on Ranges
------------------------------
+Predefined Methods on Ranges
+----------------------------
```

Comment:
Note that I chose this wording because not all of the things that follow are methods (where the current outlier is `isBoundedRange()`, though in #20885 I'm essentially proposing we retire that, at which point this label would become true.  The same applies to the bytes and strings cases:  In addition to (mostly) methods, the section defines operators and standalone `create*()` routines.  We could potentially reorder things to bring them close together and have separate sections for `Predefined [Methods|Operators|Procedures] for Strings` (say).  Or we could use a more general term like 'routines' or 'operations' (or 'functions' which I've been trying to move away from— see #17769).

Reply:
Oops, I did not notice that function; I thought there were only methods. 

For the `Arrays` and `Tuples` pages, they use the heading "Predefined Functions and Methods on _" because those APIs also have both. Would it be better to change them all to "Predefined Routines on _" unless the interface only has methods?

Reply:
Ok, I've decided to update those while I'm here. I think it looks better to have them all consistent.